docs/03-tutorials/00-eventing/evnt-03-introduction-to-jetstream.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Introduction to JetStream
+---
+
+JetStream is the 'built-in distributed persistence system' of NATS. You can read more about it [here](../../05-technical-reference/00-architecture/evnt-01-architecture.md#jetstream).
+
+## Streams and Consumers
+
+A `Stream` is a 'message store' for the published events. In Kyma, we use only one stream for all the events. You can configure the retention and delivery policies for the stream, based on the use-case.
+
+A `Consumer` is a 'view' into the stream. A kyma subscription creates one consumer for each filter specified. In Kyma we use push based consumers.
```

Comment:
```suggestion
A Consumer gives you an insight into the stream. Kyma Subscription creates one consumer for each filter specified. In Kyma we use push based consumers.
```

Reply:
Here as well: It is the view into the stream. That's why I used the single quotes.

Reply:
So it's a view into the stream, but does not give you an insight? Can you clarify how it works then?

Reply:
It's just a technical wording in my opinion. Its not an insight (I would use this term more for metrics) but this is more like `looking into the message store` if some event is meant for you